plugins/module_utils/dnac.py

```diff
@@ -202,6 +210,83 @@ def dnac_argument_spec():
     )
     return argument_spec
 
+def validate_str(item, param_spec, param_name, invalid_params):
+    item = validation.check_type_str(item)
+    if param_spec.get("length_max"):
+        if 1 <= len(item) <= param_spec.get("length_max"):
+            return item
+        else:
+            invalid_params.append(
+                "{0}:{1} : The string exceeds the allowed "
+                "range of max {2} char".format(
+                    param_name, item, param_spec.get("length_max")
+                )
+            )
+    return item
+
+def validate_int(item, param_spec, param_name, invalid_params):
+    item = validation.check_type_int(item)
+    min_value = 1
```

Comment:
Can we pass this min_value as the last argument and set 1 as default. 

def validate_int(item, param_spec, param_name, invalid_params, min_value = 1):